runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TranslationUtils.java

```diff
@@ -223,6 +223,14 @@ public Boolean call(Tuple2<TupleTag<V>, WindowedValue<?>> input) {
     }
   }
 
+  public static void rejectSplittable(DoFn<?, ?> doFn) {
+    DoFnSignature signature = DoFnSignatures.getSignature(doFn.getClass());
+
+    if (signature.processElement().isSplittable()) {
+      throw new UnsupportedOperationException(
+          String.format("SparkRunner does not support SplittableDoFn: %s", doFn));
```

Comment:
Error message nits:

 - For proper identifiers, interpolate when possible, as in `SparkRunner.class.getSimpleName()`
 - `SplittableDoFn` is not a proper identifier. It is just a `splittable DoFn`.